dashboard/config/scripts/levels/2-3 Bee Conditionals 1.level

```diff
@@ -39,5 +39,35 @@
         <block type="maze_nectar"/>
       </xml>
     </required_blocks>
+    <solution_blocks>
+      <xml>
+        <block type="when_run" movable="false">
+          <next>
+            <block type="maze_moveForward">
+              <next>
+                <block type="maze_moveForward">
+                  <next>
+                    <block type="maze_turn">
+                      <title name="DIR">turnLeft</title>
+                      <next>
+                        <block type="maze_moveForward">
```

Comment:
I see maze_moveForward appears more than once.  So, solution_blocks is not just a list of the required blocks, but is actually the correct program the student should be aiming towards?